packages/kolibri-tools/lib/i18n/crowdin.py

```diff
@@ -392,6 +392,13 @@ def download_translations(branch, project, locale_data_folder):
         code = lang_object[KEY_CROWDIN_CODE]
         locale_dir_path = local_locale_path(lang_object, locale_data_folder)
         logging.info("\tExtracting {} to {}".format(code, locale_dir_path))
+
+        #clear out existing files in langugage directory
+        for file_name in os.listdr(locale_dir_path):
```

Comment:
I think this is probably meant to be `os.listdir`?

Reply:
Yes it is, apologies. Thanks for the correction.